cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -155,24 +195,39 @@ int main(int argc, char** argv)
         std::cout << "Usage:" << std::endl;
         std::cout << "./sample_cudapoa [-m] [-h]" << std::endl;
         std::cout << "-m : Generate MSA (if not provided, generates consensus by default)" << std::endl;
+        std::cout << "-l : Perform long-read sample (if not provided, will run window-based sample by default)" << std::endl;
         std::cout << "-p : Print the MSA or consensus output to stdout" << std::endl;
         std::cout << "-h : Print help message" << std::endl;
         std::exit(0);
     }
 
     // Load input data. Each POA group is represented as a vector of strings. The sample
-    // data has many such POA groups to process, hence the data is loaded into a vector
-    // of vector of strings.
-    const std::string input_data = std::string(CUDAPOA_BENCHMARK_DATA_DIR) + "/sample-windows.txt";
+    // data for short reads has many such POA groups to process, hence the data is loaded into a vector
+    // of vector of strings. Long read sample creates one POA group.
     std::vector<std::vector<std::string>> windows;
-    parse_window_data_file(windows, input_data, 1000); // Generate windows.
-    assert(get_size(windows) > 0);
+
+    // Define upper limits for sequence size, graph size ....
+    BatchSize batch_size;
+
+    if (!long_read)
```

Comment:
optional : can you switch this to `if (long_read) else` instead of `if (!long_read) else`?

Reply:
done.